android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUISlider.java

```diff
@@ -101,6 +102,12 @@ public void processProperties(KrollDict d)
 		if (d.containsKey("leftTrackImage") || d.containsKey("rightTrackImage")) {
 			updateTrackingImages(seekBar, d);
 		}
+		if (d.containsKey(TiC.PROPERTY_TINT_COLOR)) {
+			handleSetTintColor(TiConvert.toColor(d, TiC.PROPERTY_TINT_COLOR));
+		}
+		if (d.containsKey(TiC.PROPERTY_TINT_COLOR)) {
```

Comment:
I guess this should be

```suggestion
		if (d.containsKey(TiC.PROPERTY_TRACK_TINT_COLOR)) {
```

Reply:
haha, yeah that was actually copy pasted from the TiUIProgressBar.java where it was probably also copy pasted from the tint color. I'll fix both together